charts/newrelic-k8s-metrics-adapter/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
+name: newrelic-k8s-metrics-adapter
+version: 0.1.0
+appVersion: 0.1.0
+home: https://hub.docker.com/r/newrelic/newrelic-k8s-metrics-adapter
+sources:
+  - https://github.com/newrelic/newrelic-k8s-metrics-adapter
+engine: gotpl
+icon: https://newrelic.com/assets/newrelic/source/NewRelic-logo-square.svg
+maintainers:
+  - name: douglascamata
```

Comment:
davidbrota and douglascamata are no longer part of the team

Reply:
Updated.